src/lib/components/identity/identity-manager.svelte

```diff
@@ -203,6 +238,8 @@
             searchPlaceholder="Search identities"
             loading={loading || $isAsyncLoadingIdentities}
             actionButtons={listViewButtons}
+            filters={identityFiler}
+            filterState={$identityFilterOptions}
```

Comment:
filters and filterState should be combined

Reply:
filterState was removed all together, its no longer needed